### ISSUE TITLE
Print: Entry, ":CFBundleShortVersionString", Does Not Exist (Z170X-UD5, OSX 10.12.6)

### ISSUE THREAD

**Issue by igordelorenzi** [toleda/audio_CloverALC#119]
I'm trying to fix my mic on Gigabyte Z1Z0X-UD5 and now I'm stuck on this issue. I'm also running OSX 10.12.6. Any tip?

```
$ ./audio_cloverALC-130_v0.3.command
 
Agreement
The audio_cloverALC script is for personal use only. Do not distribute
the patch, any or all of the files or the resulting patched AppleHDA.kext
for any reason without permission. The audio_cloverALC script is
provided as is and without any kind of warranty.
 
File: audio_cloverALC-130.command_v0.3
Release Mode
Password:
Print: Entry, ":CFBundleShortVersionString", Does Not Exist
Error occurred, AppleHDA.kext/Contents/Info.plist/BundleShortVersionString not found
No system files were changed
To save a Copy of this Terminal session: Terminal/Shell/Export Text As ...
logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
Deleting expired sessions...90 completed.
```

**Comment by igordelorenzi** [comment]
For some reason the original AppleHDA.kext was deleted. Once I installed it back (from backup) using KextBeat, I'm able to use the earphone! But the problem with mic still remains and it's a bit weird: the Line In (Audio line-in port) is redirecting the audio from the speaker to the mic O.o

Anyone experienced this? I'm using these devices:

![cabo-adaptador-p3-p2-combo-para-fone-e-headset-d_nq_np_326905-mlb25090607516_102016-o](https://user-images.githubusercontent.com/2574541/31142647-4af012c0-a851-11e7-91f0-8b44123f33f2.png)

![apple-earpods-original-imadxyathm6gkhyp](https://user-images.githubusercontent.com/2574541/31142736-8a5734f2-a851-11e7-9aa1-98eb919ea7fe.jpeg)

**Comment by toleda** [comment]
what ports did you connect the adapter to; specific connector, specific port, sysprefs selections?

**Comment by igordelorenzi** [comment]
Hi @toleda, I will give more info:

I'm using the front panel audio connectors like so:
![z1z0x](https://user-images.githubusercontent.com/2574541/31153105-9ede3492-a875-11e7-9b75-d552ee1ea0b9.png)
![image](https://user-images.githubusercontent.com/2574541/31153342-00b32816-a877-11e7-8071-b94391626b66.png)

`$ system_profiler SPAudioDataType` output:
```
Audio:

    Intel High Definition Audio:

      Audio ID: 1

        Speaker:

          Connection: Internal

        Line Output:

          Connection: 1/8-Inch Jack

        Line Output:

          Connection: 1/8-Inch Jack

        Internal Microphone:

          Connection: Internal

        Line Input:

          Connection: 1/8-Inch Jack

        Line Input:

          Connection: 1/8-Inch Jack

        Headphone:

          Connection: 1/8-Inch Jack

        S/PDIF Optical Digital Audio Output:

          Connection: Combination Output

        HDMI / DisplayPort Output:

          Connection: Display

    Devices:

        Built-in Input:

          Input Channels: 2
          Manufacturer: Apple Inc.
          Current SampleRate: 44100
          Transport: Built-in

        Built-in Line Input:

          Default Input Device: Yes
          Input Channels: 2
          Manufacturer: Apple Inc.
          Current SampleRate: 44100
          Transport: Built-in

        Built-in Output:

          Default Output Device: Yes
          Default System Output Device: Yes
          Manufacturer: Apple Inc.
          Output Channels: 2
          Current SampleRate: 44100
          Transport: Built-in

        Built-in Line Output:

          Manufacturer: Apple Inc.
          Output Channels: 2
          Current SampleRate: 44100
          Transport: Built-in

        Built-in Line Output:

          Manufacturer: Apple Inc.
          Output Channels: 2
          Current SampleRate: 44100
          Transport: Built-in

        Built-in Digital Output:

          Manufacturer: Apple Inc.
          Output Channels: 2
          Current SampleRate: 44100
          Transport: Built-in

        DisplayPort:

          Manufacturer: Apple Inc.
          Output Channels: 2
          Current SampleRate: 48000
          Transport: DisplayPort

        Instant On Sound Effects:

          Manufacturer: Rogue Amoeba Software, Inc.
          Output Channels: 2
          Current SampleRate: 44100
          Transport: Virtual
```

System prefs selections:
![image](https://user-images.githubusercontent.com/2574541/31153419-88079fe0-a877-11e7-87f4-38e2405810b6.png)
![image](https://user-images.githubusercontent.com/2574541/31153440-98891baa-a877-11e7-8ef0-3b975450158c.png)

Anything else I can provide to help in diagnosis?

**Comment by toleda** [comment]
"the Line In (Audio line-in port) is redirecting the audio from the speaker to the mic"
Not possible. Speaker is not connected, Headphone is connected as screenshot shows. Mic, represented as Line In, is not connected to any output; only possible with 3rd party app.

**Comment by igordelorenzi** [comment]
Hum, I have some important considerations:

- I've already tried to use the audio jacks direct on the motherboad (even separately) with no success;
- I've deducted the output of speaker is being redirected to input of mic because I saw (in sys pref panel) the bars of input level varying with the music that was playing at the moment;
- I've tested in Windows to be sure the adapter is functional :)

Is there anything I can try to figure out where I can change some config?

**Comment by toleda** [comment]
What audio jack (5 ports) equals what Sysprefs selection?
Deduction is not correct, electrically not possible; basis of misunderstanding
Adapter is working correctly, macOS/AppleHDA audio is working correctly.
Input bars moving is not speaker leaking in mic.

**Comment by igordelorenzi** [comment]
@toleda, I finally figured out what was going on. First, the patch had not been 100% applied because the installed AppleHDA was not the original. So I picked it up from the Sierra installation. Also, my input adapter was possibly defective, suffering from an "electrical crosstalk" (https://en.wikipedia.org/wiki/Crosstalk). I conclude this after getting a brand new headset and function normally.

But since everything is not perfect, now I'm going through the following situation: after some time of use, the microphone emits a very light popping sound on the headphone and stops working :(
As a palliative, I open the prefs panel and make a quick swap between the two available Line Ins and, voilà, it back to work. Do you know what could be happening?

**Comment by igordelorenzi** [comment]
UPDATE: I tried to add the

**Comment by igordelorenzi** [comment]
@toleda, you're right, it's a ground problem. I tried to plug directly on the motherboard with no issues, no noise, no sudden interruption... Now I have to figure out how to fix it or or simply forget the case connectors. Thank you a lot!
